Makefile

```diff
@@ -105,10 +106,10 @@ license: .FORCE
 format: .FORCE
 	@scripts/check_format
 
-imports: .FORCE
+imports: .build/tools/goimports
```

Comment:
Where did `.build` come from? The other targets (e.g. `docker`) are using a `build` directory.

may also be worth having a macro setup for it instead of duplicating it.